li-hadoop-plugin/src/main/resources/dependencyPattern.json

```diff
@@ -5,7 +5,7 @@
       "message": "Open source versions(org.apache.*) of artifacts are not allowed, please use linkedin artifacts(com.linkedin.*). Open source versions will be completely blocked after March 1st 2016",
```

Comment:
Good to know. In that case, I'm fine checking this in, after Alex's suggestions of
* Removing Tez from the list
* Explicitly list the Apache artifacts that are blacklisted rather than use a wildcard

are addressed.